server/tests/src/test/resources/configuration/log4j2.xml

```diff
@@ -71,7 +71,9 @@
 
     <Logger name="org.jgroups" level="INFO"/>
 
-    <Logger name="tc" level="DEBUG"/>
+    <Logger name="tc" level="INFO"/>
+
+    <Logger name="okhttp3.OkHttpClient" level="DEBUG"/>
```

Comment:
Are the changes in this file required? It seems tangentially related.

Reply:
Only the `tc` one is desired. We definitely don't want okhttp debug